homeassistant/components/mqtt/client.py

```diff
@@ -435,12 +435,13 @@ def no_more_acks() -> bool:
             """Return False if there are unprocessed ACKs."""
             return not bool(self._pending_operations)
 
-        # wait for ACK-s to be processesed (unsubscribe only)
+        # wait for ACK-s to be processed
```

Comment:
This PR doesn't change which ACK we wait for, was the comment wrong because of a previous PR?

Reply:
it was

Reply:
It was wrong